packages/dds/tree/src/test/utils.ts

```diff
@@ -109,6 +113,19 @@ export class TestTreeProvider {
         ));
     }
 
+    /**
+     * Give this {@link TestTreeProvider} the ability to summarize on demand during a test by creating a summarizer
+     * client for the container at the given index.  This can only be called when the summarizeOnDemand field is
+     * set to true.
+     * @returns void after a summary has been resolved. May be called multiple times.
+     */
+    public async manualSummarize(): Promise<void> {
+        if (TestTreeProvider.summarizeOnDemand) {
+            const summarizer = await createSummarizer(this.provider, this.containers[0]);
```

Comment:
Note that this creates another summarizer every time this is called; we only want to make one summarizer if we make one at all, which is why it would do well as an optional field as described above.

Reply:
Summarizer is now created only once in the `create` method